applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_wwcb.conf

```diff
@@ -0,0 +1,212 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
One requirement is not fulfilled here: we cannot upgrade firmware to provide security fixes. Because of that we need to rely on configuration that uses bootloader and supports DFU over HID feature reports.

Reply:
Ok, when everything else in this PR will be resolved and approved I will convert this .conf file to the configuration that uses bootloader.

Reply:
Please convert it now if possible (I would need to rereview anyway after you finish the mentioned conversion, because it will change quite a lot of things)